verticapy/sql/create.py

```diff
@@ -98,6 +105,43 @@ def create_table(
     bool
         True   if  the  table  was   successfully
         created, False otherwise.
+
+    Examples
+    --------
+    .. ipython:: python
+
+        from verticapy.utilities import create_table
+
+        # Generates the SQL needed to create the Table
+        create_table(table_name = "employees",
+                     schema = "public",
+                     dtype = {"name": "VARCHAR(60)", "salary": "FLOAT"},
+                     genSQL = True)
+
+        # Creates the table
+        create_table(table_name = "employees",
+                     schema = "public",
+                     dtype = {"name": "VARCHAR(60)", "salary": "FLOAT"})
+
+    .. code-block:: python
```

Comment:
Couldn't get sql magic to display, so just went with the html file approach. If there's a way for sql magic to work, I'll go in and fix this.

Reply:
Good solution!